pkg/storage/hot_region_storage.go

```diff
@@ -90,7 +90,7 @@ type HistoryHotRegion struct {
 
 // HotRegionStorageHandler help hot region storage get hot region info.
 type HotRegionStorageHandler interface {
-	// PackHistoryHotWriteRegions get read hot region info in HistoryHotRegion form.
+	// PackHistoryHotReadRegions get read hot region info in HistoryHotRegion form.
```

Comment:
Thanks for your work! 
btw, I noticed a missing `s` in the comment for `GetHotRegionsWriteInterval`, Could you help fix that as well?